src/resources/locales/en.json

```diff
@@ -151,6 +151,10 @@
             "bookmarkTitle": "bookmark",
             "detachWindowTitle": "detach the window",
             "fullscreenTitle": "open fullscreen mode",
+            "goTo": "Go to",
+            "goToError": "This page does not exist",
+            "goToPlaceHolder": "Go to page",
+            "goToTitle": "Change page",
```

Comment:
I think that "Change page" is not an adequate description of the action here, what about "Go to page break" instead?

Reply:
...or "Go to page" which is less technical that "page break" :)

Reply:
Fixed https://github.com/readium/readium-desktop/commit/d38073a801fbf57a96525287a026e9b0da06f196